source/resource/qx/tool/schema/compile-1-0-0.json

```diff
@@ -168,15 +168,6 @@
           },
           "parts": {
             "$ref": "#/properties/parts"
-          },
-          "minify": {
-            "$ref": "#/properties/targets"
-          },
-          "save-source-in-map": {
-            "$ref": "#/properties/targets"
-          },
-          "save-unminified": {
-            "$ref": "#/properties/targets"
```

Comment:
Wouldn't this require a change in version - it no longer is 1.0.0 if we remove a range of keys

Reply:
changing the version number of the schema is a nightmare because it effectively breaks everything and means that you must have a specific framework to compile, meaning that compiler and all packages have to be upgraded at once, or not at all

Reply:
and it will not break anything - the tags are just unsupported.

Reply:
Ok I see, glad we don#t have to change the version number for the reasons that @johnspackman mentioned.